ControlFlowArraysAndStrings/ControlFlowArraysAndStrings/Program.cs

```diff
@@ -0,0 +1,343 @@
+
+            // RemoveElementFromEnd method.
+            Console.WriteLine("\nTesting the RemoveElementFromEnd method:");
+            string[] removeApricot = RemoveElementFromEnd(fruits);
+            Print(removeApricot);
+
+            // RemoveElementFromAnywhere method.
+            Console.WriteLine("\nTesting the RemoveElementFromAnywhere method:");
+            string[] removePineapple = RemoveElementFromAnywhere(fruits, 3);
+            Print(removePineapple);
+
+            string[] removeStart = RemoveElementFromAnywhere(fruits, 0);
+            Print(removeStart);
+
+            string[] removeEnd = RemoveElementFromAnywhere(fruits, 6);
+            Print(removeEnd);
+
+            // Login method.
+            Console.WriteLine("\nTesting the Login method:");
+            Login();
+        }
+
+        /// <summary>
+        /// Sorts a copy of the string array in alphabetical order.
+        /// </summary>
+        /// <param name="array">The array to be sorted.</param>
+        /// <returns>A sorted copy of the specified array.</returns>
+        public static string[] Sort(string[] array)
+        {
+            string[] sorted = Copy(array);
+
+            for (int i = 0; i < sorted.Length - 1; i++)
+            {
+                for (int j = 0; j < sorted.Length - i - 1; j++)
+                {
+                    // Check if CompareTo() returns 1, if it does, swap positions.
+                    if (sorted[j].CompareTo(sorted[j + 1]) == 1)
+                    {
+                        string shiftedValue = sorted[j];
+                        sorted[j] = sorted[j + 1];
+                        sorted[j + 1] = shiftedValue;
+                    }
+                }
+            }
+
+            return sorted;
+        }
+
+        /// <summary>
+        /// Copies the specified array and returns a new array.
+        /// </summary>
+        /// <param name="array">The array to be copied.</param>
+        /// <returns>A copy of the specified array.</returns>
+        public static string[] Copy(string[] array)
+        {
+            string[] newArray = new string[array.Length];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                newArray[i] = array[i];
+            }
+
+            return newArray;
+        }
+
+        /// <summary>
+        /// Prints the array to the console.
+        /// </summary>
+        /// <param name="array">The array to be printed.</param>
+        public static void Print(string[] array)
+        {
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (i != array.Length - 1)
+                {
+                    Console.Write($"{array[i]}, ");
+                }
+                else
+                {
+                    Console.Write($"{array[i]}\n");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Adds a specified value to the beginning of the specified array
```

Comment:
Nit: drop "specified".  Having it twice here feels a little redundant.  Also, you're not technically adding the value to the "specified" array because you are returning a copy.